app/pages/solutions-list/filterType/manifestProvider.js

```diff
@@ -1,11 +1,15 @@
 import fs from 'fs';
 import path from 'path';
+import { ErrorContext } from 'buying-catalogue-library';
 
 export const getSolutionListManifest = (type) => {
   const filePath = `/${type}/manifest.json`;
 
   if (!fs.existsSync(path.join(__dirname, filePath))) {
-    throw new Error(`No manifest found for filter type: ${type}`);
+    throw new ErrorContext({
+      status: 404,
+      description: `No manifest found for filter type: ${type}`,
+    });
```

Comment:
This error shouldn't leak to the page.. We should log this error but return a general 500 error.

Reply:
Can we still log.error that the manifest wasn't found